Gems/ROS2/Code/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 
+find_package(sdformat13 REQUIRED)
```

Comment:
TODO: decide on sdformat version, add a comment, and decide on where to import it